docs/io/fits/index.rst

```diff
@@ -473,6 +473,42 @@ table's structure needs more information. First of all, tables can only be an
 extension HDU, not a primary. There are two kinds of FITS table extensions:
 ASCII and binary. We'll use binary table examples here.
 
+.. note::
+    
+    If you want to create a simple binary FITS table with no other HDUs,
+    you should use the Table_ interface in Astropy instead and then 
```

Comment:
It is simpler to just do
```
:class:`~astropy.table.Table`
```
This way, "stable" will properly link to the API in "stable", "latest" to "latest", and so on.